CHANGELOG.md

```diff
@@ -100,6 +100,9 @@ longer accepted automatically. Instead, the invitee can now review the invitatio
 - Updating cloud storage attached to CVAT using Azure connection string
   (<https://github.com/opencv/cvat/pull/7336>)
 
+- Can not input Korean correctly in text attributes in workspace Attribuite annotation
```

Comment:
Please, do not change CHANGELOG.md

Use ``scriv create --edit`` to create CHANGELOG fragment in `changelog.d` directory

Reply:
Thank you for introducing me to the `scriv create --edit` command. 
This was my first time using it to generate a changelog fragment in the changelog.d directory, and I must say, it's a great approach. 
I appreciate your guidance and am grateful for this new insight! :)